### DISCUSSION TITLE
Vrcx flagged as infected right before everything associated with one of my emails was hacked

### DISCUSSION THREAD

**Discussion by JekRekJek** [vrcx-team/VRCX#592] [General]
TL;DR at bottom

On the 29th I was close to getting off work when I received emails on my phone regarding steam asking about switching my email in Russian (which I do not speak) upon putting it into a translator it basically said to confirm a login from somewhere in Germany. I thought this was weird, and did not click on anything in the email. Soon after that one another one came in stating my email had been changed, and so I began to worry because I had quite a few things on my steam I'd prefer to keep. As soon as I got off work I hurried home and turned on my pc to find out at the same time as I was receiving these emails my computer flagged vrcx's application and something downloaded in the folder with it as trojans in the windows security section of the settings. Soon after I found myself unable to login to steam, Microsoft, and that I had been logged out of my email on my pc because of "suspicious activity" from an app.
This all happened the day after I downloaded vrcx from this very Github and let it be downloaded even with my pc's warnings, because it had done nothing wrong to my device in the past.

TL;DR 
Trusted app because it had done no harm in the past, newest update lost me my steam account for 3 days and had to locate a gift card I used 3 years ago to get my account back. My Microsoft was lost, and I had to repurchase Minecraft. This was all due to the newest update having some sort of trojan in it.

![image_2023-07-02_234202364](https://github.com/vrcx-team/VRCX/assets/138432326/609555bf-91bf-4cbb-ba65-8f2fa93db3a6)

**Comment by affectioned** [comment]
Are you sure you haven't downloaded anything else as well in the past 3 days before downloading VRCX? What type of your protection does your microsoft and steam account have? (do you have mobile authenticator apps for both?)

**Comment by Natsumi-sama** [comment]
Please read [this issue](https://github.com/vrcx-team/VRCX/issues/568) where we go into detail as to why this is a false positive.

https://en.wikipedia.org/wiki/Correlation_does_not_imply_causation

**Comment by OctoMiku01** [comment]
I had Windows flag it too, whitelisted it and nothing happened since days. Must be something else.

**Comment by Myrkie** [comment]
a follow up to my own issues and reports, even microsofts OWN datacab files they ask for us to provide full of microsoft defender logs get reported as malware by cloud protection, here is the latest submission of our vrcx executible plus microsofts log files from defender their own files get reported as a virus. AI detection is flawless with no issues what so ever /s


![image](https://github.com/vrcx-team/VRCX/assets/20288698/d361c7d9-6953-4dc3-bc8c-30889bd17415)

![image](https://github.com/vrcx-team/VRCX/assets/20288698/196b5264-99a1-43d2-a848-b7f0aa05a56a)

**Comment by Myrkie** [comment]
Response has been received from Microsoft again, signature for the detected VRCX version has been removed from detection's database, this doesn't mean it wont happen again in the future and probability is very high thanks to Microsoft's increased use of unstable AI Detection's if anymore issues arise go through the message requirement outlined in https://github.com/vrcx-team/VRCX/issues/586#issuecomment-1610917592
